### DISCUSSION TITLE
Orca stops working

### DISCUSSION THREAD

**Discussion by tardisbus** [Guilouz/Creality-Helper-Script-Wiki#528] [Issues]
Have the K1 rooted, Fluidd, mainsail etc.. But after adding a Bambu A1mini to Orca, now K1 wont connect. 
![2024-04-22 (2)](https://github.com/Guilouz/Creality-K1-Series/assets/167808220/ca64100f-7510-47d1-8416-168cb26f4404)
![2024-04-22](https://github.com/Guilouz/Creality-K1-Series/assets/167808220/accc3b4e-8960-4d1a-ae5c-bf7a7bff70e7)
No idea how to get this working again

**Comment by onkel172** [comment]
no Ports are defined, 192.168.0.85:4408 for fluidd and 192.168.0.85:4409 for mainsail

**Reply by tardisbus** [reply]
Yep. That worked. Forgot about the port. Thank you
